src/components/Nav/_mixins.scss

```diff
@@ -194,7 +184,8 @@ $nav__icon__spacing: $carbon--spacing-05 + $nav__spacing;
 
         position: relative;
         display: flex;
-        padding: $carbon--spacing-02;
+        height: 32px;
+        padding: 0.3125rem $carbon--spacing-02 $carbon--spacing-02 2.3rem;
```

Comment:
is there a [spacing variable](https://www.carbondesignsystem.com/guidelines/spacing/overview/) you can use here?

Reply:
Done!